app/models/course.rb

```diff
@@ -168,6 +168,17 @@ def duplication_manifest
     ]
   end
 
+  # TODO: Need to replace this with an assessment settings adapter in future
+  # Course setting to enable public test cases output
+  def show_public_test_cases_output
```

Comment:
Any reason why this is placed under `Course` settings instead of `Assessment` settings?

Reply:
Oh ok saw it in the previous conversation, no assessment adapter yet.

Reply:
@weiqingtoh Yeah actually, After discussion with @allenwq, we noticed that there wasn't an assessment settings adapter. Thats why it was placed under course setting.